### ISSUE TITLE
Inconsistent maxYear, maxMonth

### ISSUE THREAD

**Issue by mcroba** [Pikaday/Pikaday#342]
Hi,

There is a bug in your setMaxDate function. The maxYear and the maxMonth are not updated (it is in the setMinDate) and then you could be stuck in the past (with the navigation).

``` javascript
/**
 * change the minDate
 */
setMinDate: function(value)
{
    setToStartOfDay(value);
    this._o.minDate = value;
    this._o.minYear  = value.getFullYear();
    this._o.minMonth = value.getMonth();
},

/**
 * change the maxDate
 */
setMaxDate: function(value)
{
    this._o.maxDate = value;
},
```

Thanks in advance for the fix ^^
Best regards,
Sylvain

**Comment by nathancahill** [comment]
This is fixed by #317

**Comment by wojciechczerniak** [comment]
It's already fixed. Closing the issue.
